### PR TITLE
chore: update THIRD_PARTY_LICENSES.txt to prepare for v0.1.0

### LINKED ISSUE #126: Update THIRD_PARTY_LICENSES.txt for SQLAlchemy, CUE and Souffle
This ticket documents the task to update the THIRD_PARTY_LICENSES.txt file with the new third party dependencies for the open source v0.1.0 release.

### PR DESCRIPTION
**Pull request by tromai**
Closes #126

### CODE REVIEW

File: THIRD_PARTY_LICENSES.txt [hunk 1]
```diff
@@ -1107,55 +1107,55 @@ SOFTWARE.
 Mozilla Public License
 Version 2.0
 1. Definitions
-1.1. “Contributor”
+1.1. â€œContributorâ€
```
Comment (behnazh-w) [inline THIRD_PARTY_LICENSES.txt:1110]: I wonder if these characters where intended?

Comment (tromai) [reply THIRD_PARTY_LICENSES.txt:1110]: Hmmm, they are not intended. I will have a look through to address all of the similar issues.

Comment (tromai) [reply THIRD_PARTY_LICENSES.txt:1110]: This regex can be used to find weird symbols such as this in the whole file `(?![©“”’öíéáñéø简体中文])[^\x00-\x7F]+`. Note that all symbols in `(?![<exclude_symbols_here])` will be excluded.

Comment (tromai) [reply THIRD_PARTY_LICENSES.txt]: Fixed in https://github.com/oracle-samples/macaron/pull/127/commits/b9abf776ebd3ba9ab6944cf6db3779f1698caa52

**Review by behnazh-w — APPROVED** [review]

**Pull request merged**